docs/source/models/gbd2017/causes/cause_model_template.rst

```diff
@@ -91,125 +82,193 @@ Disease Description
 .. admonition:: Todo for template development
 
-  - Are the sections in a good order? In `Pull Request 91`_, @Lu said:
+  Decide on section names and overall structure.
 
-      The template looks good to me. I was putting the model assumptions and
-      limitations section right after the cause model diagram. But I think
-      this order makes more sense.
+  **Question:** Are the sections in a good order?
 
-    And @Yongquan said:
+  In `Pull Request 91`_, @Lu said:
 
-      Model assumptions and/or limitations can be mentioned in summary disease
-      model description and fully explained in Model Assumptions and
-      Limitations section.
+    The template looks good to me. I was putting the model assumptions and
+    limitations section right after the cause model diagram. But I think
+    this order makes more sense.
 
-    Whereas @James said:
+    ("This order" referring to: `Cause Model Diagram`, `Data Description`,
+    `Model Assumptions and Limitations`.)
 
-      I think the restrictions in this section should move up to the GBD Modeling section.
+  And @Yongquan said:
 
-      Also, perhaps we should have a section following the GBD section called
-      `vivarium modeling strategy` which would include the scope and the
-      restrictions we apply to the model (which might be different than GBD's
-      restrictions). To serve as a narrative description accompanying the cause
-      model diagram and data tables.
+    Model assumptions and/or limitations can be mentioned in summary disease
+    model description and fully explained in Model Assumptions and
+    Limitations section.
 
-    On the other hand, @Kiran said:
+  Whereas @James said:
 
-      I am good with this structure. But, we have to make changes to the
-      causes that are finished. Also, for restrictions I like it under
-      Assumptions and Limitations section. We can add subsections if there are
-      different types of restrictions.
+    I think the restrictions in this section (`Model Assumptions and
+    Limitations`) should move up to the GBD Modeling section.
 
-    Do we have examples of restrictions we would apply that are different from
-    GBD restrictions?
+    Also, perhaps we should have a section following the GBD section called
+    `vivarium modeling strategy` which would include the scope and the
+    restrictions we apply to the model (which might be different than GBD's
+    restrictions). To serve as a narrative description accompanying the cause
+    model diagram and data tables.
 
-Restrictions
-++++++++++++
+  On the other hand, @Kiran said:
 
-Scope
-+++++
+    I am good with this structure. But, we have to make changes to the
+    causes that are finished. Also, for restrictions I like it under
+    Assumptions and Limitations section. We can add subsections if there are
+    different types of restrictions.
 
-Validation Criteria
--------------------
+  **Question:** Do we have examples of restrictions we would apply that are
```

Comment:
A cause example would be something where the mortality restrictions are wider than the morbidity restrictions, meaning we have deaths due to a cause that GBD says there is no prevalence for.